packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -29,6 +29,14 @@ const nextConfig = {
   // Enable React Strict Mode
   reactStrictMode: true,
 
+  // enable image provider domains and device sizes
+  images: {
+    domains: ['cm.jss.localhost'],
```

Comment:
I think that it should be dynamic, you need to take value from `temp/config.js`, and have something for default

Reply:
Yes, should use `jssConfig.sitecoreApiHost` (domain only). Not sure we need a default?

Reply:
Looks like this was updated to the literal string, but I meant the imported value (from './src/temp/config'). We'd want the domain only (no scheme).